data/sql/updates/pending_db_world/rev_1734664149862386300.sql

```diff
@@ -0,0 +1,2 @@
+-- loot table for item 22568 Sealed Craftsman's Writ
+UPDATE `item_template` SET `flagsCustom` = 2 WHERE `entry` IN (22600, 22601, 22602, 22603, 22604, 22605, 22606, 22607, 22608, 22609, 22610, 22611, 22612, 22613, 22614, 22615, 22616, 22617, 22618, 22620, 22621, 22622, 22623, 22624);
```

Comment:
Please update the query to only set / unset the bits in the flag field that you want to change instead if writing the full value.

Reply:
e.g.

`flagsCustom` = `flagsCustom` | 2

To set the bit 2.